packages/trampoline/trampoline.0.1.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "A trampoline implementation"
+description: "A trampoline implementationtion"
```

Comment:
Can you make the `synopsis` and the `description` more descriptive?

Reply:
I'm working on it.

Reply:
@mseri The ocaml ci page seems to be unavailable. Did my test crash the test server?

Reply:
@QAhell there was an unrelated hardware failure, not your fault at all! :-) https://github.com/ocaml/infrastructure/issues/51

Just restarted some failed CI jobs and should be ready to merge this package soon.